addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -301,7 +305,7 @@ async function discuss_channel_messages(request) {
             MailMessage.browse(messages.map((message) => message.id)),
             makeKwArgs({ for_current_user: true })
         ).get_result(),
-        messages: mailDataHelpers.Store.many_ids(messages),
+        messages: mailDataHelpers.Store.many(messages)._get_id(),
```

Comment:
this is just messages.map(m => m.id) or something like that, we don't need to use store there.